resources/language/resource.language.sv_se/strings.po

```diff
@@ -614,7 +614,7 @@ msgstr "Populära"
 
 msgctxt "#30364"
 msgid "Latest news"
-msgstr "Senaste nyhetsprogram"
+msgstr "Senaste nytt"
```

Comment:
@sopor I don't think this is correct. These are actually the most recent episodes of news shows.

Reply:
The content seems to come from: goto  svtplay.se" / Select "Program" / Select "Nyheter" scroll down and select "Senaste". 
So I think the correct would be "Senaste nyheter" or  as it is now, "Senaste nyhetsprogram".